### DISCUSSION TITLE
how to use physics (ammo.js) in current version of Tresjs

### DISCUSSION THREAD

**Discussion by rmrbytes** [Tresjs/tres#523] [Q&A]
I read that the physics integration into this wonderful tool is underway. Meanwhile how does one apply a physics method (say applyForce, setVelocity etc) to a Tres object.

Thanks

**Comment by andretchen0** [comment]
Sorry, I haven't used Ammo, and I don't want to lead you down the wrong path.

Just chiming in to say that (in case you haven't) you might have better luck with "how to" questions if you ask on the TresJS Discord.

**Reply by rmrbytes** [reply]
Thanks @andretchen0 , I will pursue it there
